resources/assets/lib/report-form.coffee

```diff
@@ -31,9 +31,7 @@ options = [
 ]
 
 export class ReportForm extends PureComponent
-  constructor: (props) ->
```

Comment:
pass customization options in through `props` and compose specialized variations of `ReportForm` instead of making it differentiate between score and user

https://reactjs.org/docs/composition-vs-inheritance.html#specialization